test/src/server.py

```diff
@@ -2128,3 +2139,13 @@ def setup_test_iface_ip_net(self: 'LoadGen'):
         """
         self.exec(f'sudo ip link set {self.test_iface} up')
         self.exec(f'sudo ip address add {self.test_iface_ip_net} dev {self.test_iface}')
+
+
+    def run_iperf_client(self, ipt: "IPerfTest"):
```

Comment:
we also need and use a kill iperf client function to be called initially to cleanup processes from potential earlier (failed) runs